tools/build.py

```diff
@@ -178,7 +192,29 @@ def Build(self, project_definition):
       raise ValueError('Unsupported download URL: {0:s}.'.format(
           project_definition.download_url))
 
-    return self._BuildProject(download_helper_object, project_definition)
+    # TODO: implement
```

Comment:
Remove todo, or clarify what's supposed to be implemented.

Reply:
This hasn't been addressed.

Reply:
refactor marker, did not see this previously due to github UI